aws-nitro-enclaves-k8s-ds.yaml

```diff
@@ -33,10 +33,8 @@ spec:
       containers:
       - name: aws-nitro-enclaves-k8s-dp
         env:
-        - name: LOG_LEVEL
-          value: "INFO"
         # max 4 devices supported
-        - name: MAX_DEVICES_PER_NODE
+        - name: MAX_ENCLAVES_PER_NODE
```

Comment:
This renaming should probably be wrapped into the previous commit, along with a lot of the refactoring in the device plugins below (e.g. we are removing the `deviceName` argument of `generateDeviceID` in the previous commit, just to reintroduce it here). Having some of that back and forth removed will make it a lot easier to reason about the past when looking at the commit history in 5 years from now.

Reply:
done